geo/benches/intersection.rs

```diff
@@ -144,6 +144,97 @@ fn point_triangle_intersection(c: &mut Criterion) {
     });
 }
 
+fn linestring_polygon_intersection(c: &mut Criterion) {
+    use geo::{coord, line_string, LineString, Polygon, Rect};
+    c.bench_function("LineString above Polygon", |bencher| {
+        let ls = line_string![
```

Comment:
This is just a minor personal preference, so take it or leave it, but if you use the `geo::wkt!` macro for constructing geometries in tests, not only will it be shorter, but also, I can then easily copy/paste it into JTSTestBuilder or some other tool to visually inspect the test cases. 

(It doesn't work for non-standard types like Rect/Triangle, though maybe it should?)